admin_guide/scoped_tokens.adoc

```diff
@@ -11,39 +11,43 @@
 toc::[]
 
 == Overview
-A user may wish to give another entity the power to act as him, but only in a limited way.
-For example, a project admin may wish to delegate the power to create pods.  One way to do 
-this is to create a scoped token.
-
-A scoped token, is a token that identifies as a given user, but is limited to certain actions 
-by its scope.  Right now, only a cluster-admin can create scoped tokens.
+A user may wish to give another entity the power to act as him or her, but only
```

Comment:
I'd avoid this him/her gender dichotomy, a good rule in general for technical writing.

Suggest rephrase with "they":  A user may want to give another entity the same power to act that they have, but only in a limited way.